src/nfv_test_api/util.py

```diff
@@ -206,15 +210,13 @@ def setup_namespaces():
     for ns in cfg.namespaces.values():
         for intf in ns.interfaces:
             if intf.mac in mac_lookup:
-                if mac_lookup[intf.mac]["namespace"] is not None:
-                    LOGGER.error(f"Cannot move interface {mac_lookup[intf.mac]['name']} to another namespace.")
-                    continue
-
```

Comment:
@bartv I assumed this is no longer relevant, since we now can move an interface between network namespaces, but I'm not entirely sure.

Reply:
indeed